Tests/AESDecrypterTests.swift

```diff
@@ -59,12 +59,37 @@ class AESDecrypterTests: RSACryptoTestCase {
 
     /// Tests the `AES` decryption implementation for AES_256_CBC_HMAC_SHA_512 with the test data provided in the [RFC-7518](https://tools.ietf.org/html/rfc7518#appendix-B.3).
     func testDecryptingA256CBCHS512() {
-        let keyData = "00 01 02 03 04 05 06 07 08 09 0a 0b 0c 0d 0e 0f 10 11 12 13 14 15 16 17 18 19 1a 1b 1c 1d 1e 1f 20 21 22 23 24 25 26 27 28 29 2a 2b 2c 2d 2e 2f 30 31 32 33 34 35 36 37 38 39 3a 3b 3c 3d 3e 3f".hexadecimalToData()
-        let additionalAuthenticatedData = "54 68 65 20 73 65 63 6f 6e 64 20 70 72 69 6e 63 69 70 6c 65 20 6f 66 20 41 75 67 75 73 74 65 20 4b 65 72 63 6b 68 6f 66 66 73".hexadecimalToData()
+        let keyData = """
+        00 01 02 03 04 05 06 07 08 09 0a 0b 0c 0d 0e 0f 10 11 12 13 14 15 16 \
+        17 18 19 1a 1b 1c 1d 1e 1f 20 21 22 23 24 25 26 27 28 29 2a 2b 2c 2d \
+        2e 2f 30 31 32 33 34 35 36 37 38 39 3a 3b 3c 3d 3e 3f
+        """.hexadecimalToData()
```

Comment:
An alternative to having these inline would be to use [NSDataAsset]. @daniel-mohemian what do you think?


[NSDataAsset]: https://nshipster.com/nsdataasset/

Reply:
I think that would be cool. 👍 Probably also a lot of work, would it make sense to do it separately, what do you think?

Reply:
Yeah, I think so too. Another PR, then. I just wanted to mention it.